app/src/main/cpp/adpf_manager.cpp

```diff
@@ -116,12 +123,15 @@ bool ADPFManager::InitializePowerManager() {
 
 // Retrieve current thermal headroom using JNI call.
 float ADPFManager::UpdateThermalStatusHeadRoom() {
```

Comment:
Need to logic for checking NaN value.

Reply:
This is a pass-through, the sample will display NaN on the UI.
But we can discuss this and submit a separate PR, this PR is mainly for PerformanceHintAPI, the minor change affecting Thermal is because I change to std::chrono clock for standard